sdk/storage/Azure.Storage.Blobs/src/autorest.md

```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/0798e9c8eeae475849d07d7fe17fd1abcffe8aad/specification/storage/data-plane/Microsoft.BlobStorage/stable/2025-07-05/blob.json
+    - https://raw.githubusercontent.com/nickliu-msft/azure-rest-api-specs/e80abcc014fec4fad056232a3a38d221293e0ed1/specification/storage/data-plane/Microsoft.BlobStorage/stable/2025-11-05/blob.json
```

Comment:
We need to merge the swagger PR for this feature and change this line to point at the merged swagger before we merged this .NET PR.